packages/core/data-transfer/src/engine/__tests__/engine.test.ts

```diff
@@ -20,6 +21,27 @@ import {
 
 const getMockSourceStream = (data: Iterable<unknown>) => Readable.from(data);
 
+const defaultLinksData: Array<ILink> = [
+  {
+    kind: 'relation.basic',
+    relation: 'oneToOne',
+    left: { type: 'api::foo.foo', ref: 1, field: 'foo' },
+    right: { type: 'api::bar.bar', ref: 2, field: 'bar' },
+  },
+  {
+    kind: 'relation.basic',
+    relation: 'oneToMany',
+    left: { type: 'api::foo.foo', ref: 1, field: 'foos' },
+    right: { type: 'api::bar.bar', ref: 2, field: 'bar' },
+  },
+  {
+    kind: 'relation.basic',
+    relation: 'oneToMany',
+    left: { type: 'basic.foo', field: 'foo', ref: 1 },
+    right: { type: 'api::foo.foo', ref: 1 },
+  },
+];
```

Comment:
- Should line 39 be manyToOne?
- Not sure if line 34 needs to be singular, or if line 35 needs to be plural.  

I'm not a typescript person, just noticing some differences :smile: